druid/examples/anim.rs

```diff
@@ -74,7 +83,6 @@ pub fn main() {
             .with_placeholder("You spin me right round..."),
     );
     AppLauncher::with_window(window)
-        .use_simple_logger()
```

Comment:
I think we should use logging in all examples; it's important that people have logging enabled during development, so that we can do things like print warnings about incorrect layout.

Reply:
I don't really mind, I thought that nobody would really use the examples for real development, but I guess people copy paste examples all the time. I will also put it back for the identity in this PR as well then.